modules/service/src/main/scala/lucuma/odb/service/ObservingModeServices.scala

```diff
@@ -22,7 +21,7 @@ sealed trait ObservingModeServices[F[_]] {
 
   def selectObservingMode(
     which: List[(Observation.Id, ObservingModeType)]
-  )(using Transaction[F], SuperUserAccess): F[Map[Observation.Id, SourceProfile => ObservingMode]]
+  )(using Transaction[F], SuperUserAccess): F[Map[Observation.Id, ObservingMode]]
```

Comment:
No more `SourceProfile => ObservingMode`.  It's just an `ObservingMode` now.  This makes the `reconcile` concept moot as well so I could also remove that.